### ISSUE TITLE
Sentric key?

### ISSUE THREAD

**Issue by dondv** [omrilotan/tagmanager-sentry#1]
Great job with this. 

What is the Sentry Key and where do I find it?

Is it the public key or the project identifier.

Thanks

Br

**Comment by omrilotan** [comment]
You can find your key in your project's "Client Keys" page.
I will add it to readme as soon as I can
![](https://user-images.githubusercontent.com/516342/78922573-720c0300-7a9f-11ea-84c8-bd423e609265.png)

**Comment by omrilotan** [comment]
Sorry, @dondv will this instruction help you? (Addressed in #2)

---

Find your project ID and public key [in you settings page](https://sentry.io/settings):
```
─ projects
  └─ <PROJECT_NAME>
     └─ Client Keys (DSN)
        └─ Configure (button)
           ├─ Public Key (scroll down)
           └─ Project ID
```

**Comment by omrilotan** [comment]
Resolved in #2

**Comment by dondv** [comment]
Thanks!
